client/auth/lightbox/iframe_channel.js

```diff
@@ -54,8 +54,7 @@ define([
      * Send a message to the iframe.
      *
      * @method send
-     * @param {String} command
-     * Message to send.
+     * @param {String} command Message to send.
```

Comment:
I have no idea why, but it seems like YUIDoc was mangling this unless i merged onto a single line:
<img width="352" alt="iframechannel_-_fxa-relier-client" src="https://cloud.githubusercontent.com/assets/557895/9616257/f77bc70e-50b1-11e5-9ec2-6064d6b8623d.png">